var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -56,6 +56,7 @@ class Openblas(MakefilePackage):
     #  This patch is in a pull request to OpenBLAS that has not been handled
     #  https://github.com/xianyi/OpenBLAS/pull/915
     patch('openblas_icc.patch', when='%intel')
+    patch('openblas_icc_openmp.patch', when='%intel@18.0:')
```

Comment:
You should just use it for 16+. Both work but -openmp spits out thousands of lines of depreciation warnings.